owncloudApp/src/main/java/com/owncloud/android/presentation/sharing/shares/PublicShareDialogFragment.kt

```diff
@@ -310,6 +326,12 @@ class PublicShareDialogFragment : DialogFragment() {
         }
     }
 
+    private fun initPasswordChangeInputListener() {
+        binding.shareViaLinkPasswordValue.doOnTextChanged { text, _, _, _ ->
+            updateRequirements(text.toString(), binding.shareViaLinkPasswordPolicy)
```

Comment:
This method, which is called first the first time a change is made in the text inside the `EditText`, is the one that makes the text for the requirements visible. It should be visible before writing as well! No surprises for the user once they start writing... But I think you can solve this by handling the requirements visibility in different TextViews as I suggested in a previous comment

Reply:
Done

Reply:
Finally it was decided that the requirements are shown after the text is changed, so you can leave it like this 👍

Reply:
Done